comp/trace/agent/run.go

```diff
@@ -86,7 +87,12 @@ func runAgentSidekicks(ctx context.Context, cfg config.Component, wmeta workload
 
 	if coreconfig.IsRemoteConfigEnabled(coreconfig.Datadog) {
 		// Auth tokens are handled by the rcClient
-		rcClient, err := rc.NewAgentGRPCConfigFetcher()
+		ipcAddress, err := coreconfig.GetIPCAddress()
+		if err != nil {
```

Comment:
This error should be reported as Startup Error via `telemetryCollector.SendStartupError(telemetry.CantCreateRCCLient, err)`

Reply:
Done